dev_info/build.html

```diff
@@ -0,0 +1,31 @@
+﻿<!DOCTYPE html>
+
+<link rel="stylesheet" href="../locales/help_enu/styles.css" type="text/css" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta charset="utf-8" />
+<title>Building GEDKeeper</title>
+</head>
+<body>
+<h1>How to build GEDKeeper</h1>
+<h2>Microsoft Windows</h2>
+<h3>Dependencies</h3>
+<p>The following is a list of required software to build and test:
+<ul>
+<li>Microsoft <a href="https://dotnet.microsoft.com/">.NET Framework</a> 4.0.30319</li>
+<li>Microsoft <a href="https://visualstudio.microsoft.com/">Visual Studio</a> 2017 or later</li>
+<li>Nullsoft Scriptable Install System (<a href="https://sourceforge.net/projects/nsis/">NSIS</a>) 3.04</li>
+<li><a href="https://sourceforge.net/projects/sevenzip/">7-zip</a></li>
+<li><a href="https://github.com/nunit">NUnit</a> 2.6.4 (unit testing framework for .NET)</li>
+</ul>
+</p>
+<h3>Execution</h3>
+<p>Run the executable <kbd>build_all_mswin.x86.cmd</kbd> from the command line.</p>
+<p>Optionally use the parameter &quot;Release&quot; for a release build (the default is a debug build). Testing is not done with a release build.</p>
+
+<h2>Linux</h2>
+<p>Sorry, not yet implemented.</p>
```

Comment:
I thought it is.  :-)  @Serg-Norseman, what do you think about?

Reply:
That what is? A Linux build? There is support for Mono, but if there is a native Linux build then I am blind to seeing it.

Reply:
I think **native** (ELF) executable is impossible, since GEDKeeper is written in C#.  I don't know what format Mono generates and how you execute that then.  @Serg-Norseman knows  :-D

If we don't count "having Mono executable" as Linux supporting, then yes -- Linux is unsupported platform.

Reply:
Oh, men!  I forgot about this, but "a .NET application is made up of instructions in a language called CIL"[1].  Well, GEDKeeper.exe isn't a native Microsoft Windows program either  :-D  It's a CLR who supports a target system, not a program.  Therefore, if we "don't support" LInux, we don't support Microsoft Windows either.

[1] https://stackoverflow.com/a/5188580

Reply:
Because we decided to add notes here specific to Linux-based systems, I am closing this review.